src/commands/diff/index.ts

```diff
@@ -117,6 +137,12 @@ export default class Diff extends Base {
         }
         Object.values(matchesBySdk).forEach((matches) => {
             matches.forEach((match) => {
+                const aliasedName = aliasMap[match.name]
+                if (match.isUnknown && aliasedName) {
+                    match.alias = match.name
```

Comment:
do you think we should avoid mutation here? Technically these lists are being passed around in series so it shouldn't matter, but it could lead to something unexpected later maybe?